### PR TITLE
Fix #4058 - add SpliceAI link

### LINKED ISSUE #4058: Add a linkout to SpliceAI-lookup from variant page, splice AI prediction
https://spliceailookup.broadinstitute.org

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Adds a link that in particular @47KW wanted. Aside from colouring the values, I think it will be useful for 1) compute on slightly larger indels 2) includes pangolin scores.

Links from here:
Screenshot 2023-08-28 at 15 04 31

to here:
Screenshot 2023-08-28 at 15 06 07

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 1]
```diff
@@ -288,7 +288,7 @@
           <span class="float-end">{{ variant.spidex_human }}</span>
         </li>
         <li class="list-group-item">
-          <a href="https://doi.org/10.1016/j.cell.2018.12.015" target="_blank">SpliceAI</a> <a href="https://github.com/Illumina/SpliceAI" target="_blank">DS max</a>
+          <a href="https://doi.org/10.1016/j.cell.2018.12.015" target="_blank">SpliceAI</a> (<a href="{{ variant.spliceai_link }}">Lookup</a>) <a href="https://github.com/Illumina/SpliceAI" target="_blank">DS max</a>
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:291]: Nice! Perhaps a couple of links here are a bit redundant? If you open the github repo page with the third link it already points you to the paper, so perhaps remove the first link? 
 would also make the new link open in a new window. I'll add a suggestion!

Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:291]:
```suggestion
          SpliceAI (<a href="{{ variant.spliceai_link }}" target="_blank" rel="noopener">Lookup</a>) <a href="https://github.com/Illumina/SpliceAI" target="_blank" rel="noopener">DS max</a>
```

**Review by northwestwitch — APPROVED** [review]
Nice, I'm sure it's going to be appreciated! 👍🏻

File: scout/server/blueprints/variant/templates/variant/variant_details.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/variant_details.html]: Actually, we could just switch the links then and skip the lookup.. 😅

**Comment by dnil** [comment]
😆 Tyvärr, du måste göra det själv, som du skrev:

**Pull request merged**